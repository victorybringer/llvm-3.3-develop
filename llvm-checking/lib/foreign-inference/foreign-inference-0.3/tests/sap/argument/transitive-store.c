struct S {
  int *p1;
  int *p2;
};

struct T {
  int *t1;
  struct S *t2;
};

void g(int *x, struct S *s) {
  s->p1 = x;
}

void f(struct T *t, int *x) {
  g(x, t->t2);
}
