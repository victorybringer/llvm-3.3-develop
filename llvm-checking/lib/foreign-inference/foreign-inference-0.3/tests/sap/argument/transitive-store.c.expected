fromList [ (("f", "x"), fromList [(0, "%struct.T*", [AccessDeref,AccessField 1, AccessDeref, AccessField 0])])
         , (("g", "x"), fromList [(1, "%struct.S*", [AccessDeref, AccessField 0])])
         ]